### ISSUE TITLE
Accessibility of sidebar toggle

### ISSUE THREAD

**Issue by zersiax** [poole/lanyon#89]
Currently, at least on the demo site, the button for toggling the sidebar in this theme cannot be seen nor interacted with by people using a screenreader to access the website the theme is currently being used on. This means the menu that is normally contained within this sidebar is not reachable by these users and therefore the site cannot be used in its entirety. I would like to ask for this to be fixed, so sites using this theme will not suffer from this rather serious accessibility problem.
